README.md

```diff
@@ -3,17 +3,21 @@
 Provides support for parsing and emitting Yaml documents.
 
 This package includes the [full libyaml C library version 0.1.5 by Kirill
-Simonov](http://pyyaml.org/wiki/LibYAML) in the package so you
```

Comment:
Due to the reflowing of the lines, it's difficult to tell if there was actually a content change here. Can you point out what has actually changed?